### DISCUSSION TITLE
Does OpenAM comes with an integrated FIDO server to support WebauthN ?

### DISCUSSION THREAD

**Discussion by amitt0488** [OpenIdentityPlatform/OpenAM#647] [General]
I see webauthn is supported in openAM 
https://github.com/OpenIdentityPlatform/OpenAM/wiki/How-to-Setup-WebAuthn-Authentication-in-OpenAM  

But, I don't see any way to configure a external FIDO server to achieve this in openAM and the only way looks like is to write custom modules for it.
Does that mean that OpenAM comes with an integrated FIDO server to support WebauthN ?

**Comment by vharseko** [comment]
@maximthomas

**Comment by maximthomas** [comment]
@amitt0488 OpenAM acts as FIDO server, so you can register keys and use registered keys for authenticateion.
